cwms-data-api/src/main/java/cwms/cda/api/ForecastTimeseriesController.java

```diff
@@ -0,0 +1,72 @@
+package cwms.cda.api;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import cwms.cda.data.dao.JooqDao;
+import io.javalin.apibuilder.CrudHandler;
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+import java.util.logging.Logger;
+
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.cda.api.Controllers.RESULTS;
+import static cwms.cda.api.Controllers.SIZE;
+
+public class ForecastTimeseriesController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(ForecastSpecController.class.getName());
+
+    public static final String TAG = "ForecastTimeseries";
```

Comment:
I think all the forecast endpoints should be grouped together. They can share the tag "Forecast"

Reply:
done